CHANGELOG.md

```diff
@@ -76,36 +77,6 @@
 - PR #1319 CSV Reader: Use column wrapper for gdf_column output alloc/dealloc
 - PR #1376 Change series quantile default to linear
 - PR #1399 Replace CFFI bindings for NVTX functions with Cython bindings
-- PR #1407 Rename and cleanup of `gdf_table` to `device_table`
```

Comment:
Why does this PR delete from the changelog?

Reply:
Automatic action while rebasing on 0.8 . Fixing that.